test/helpers/time_helper_test.rb

```diff
@@ -14,6 +15,7 @@ class TimeHelperTest < ActionView::TestCase
   end
```

Comment:
Does `travel_to` need a block to unravel itself at the end of the test, or is it locally scoped?